max-chelsea/starter-code/public/admin.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Lab 10</title>
+    <link rel="stylesheet" href="/vendor/styles/normalize.css">
+    <link rel="stylesheet" href="/styles/icons.css">
+    <link rel="stylesheet" href="/styles/base.css">
+    <link rel="stylesheet" href="/styles/layout.css">
+    <link rel="stylesheet" href="/styles/modules.css">
+    <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+    <!-- Done: Setup a template for each row of the Author Stats section of this page: -->
+    <script id = "author-template" type = "text/x-handlebars-template">
+        <li id="authorName">"{{ author }}"</li>
+        <li id="authorWords">"{{ wordCount }}"</li>
+    </script>
+
+  </head>
+  <body>
+    <header class="site-header clearfix">
+      <section class="heading-group-h1 pull-left">
+        <h1>
+          Kilovolt Blog!
+        </h1>
+        <h2>The shocking power of static pages</h2>
+      </section>
+      <nav class="main-nav">
+        <div class="icon-menu"></div>
+        <ul>
+          <li class="tab"><a href="index.html" class="icon-home"></a>Blog</li>
+          <li class="tab"><a href="stats.html"></a>Stats</li>
+          <li class="tab"><a href="new.html">New Article</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="clearfix">
+      <h1>Blog Stats</h1>
+      <section id="blog-stats">
+        <dt>Total articles:</dt> <dd class="articles"></dd>
+        <dt>Total words:</dt> <dd class="words"></dd>
+        <h2>Author Stats</h2>
+        <p><em>Details on who is writing, and how much writing they are doing...</em></p>
+        <ul class="author-stats">
+        </ul>
+      </section>
+    </main>
+    <footer>
+      &copy; 2017 Code Fellows | Static Blog | Happy Ipsum!
+    </footer>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="/vendor/scripts/handlebars.js"></script>
+    <script src="/scripts/article.js"></script>
+    <script src="/scripts/articleView.js"></script>
+    <!-- DONE: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <script>
+      // DONE: After we fetch and load all the articles, what view function do we want to run next?
+      // Let's pass in a callback function, to determine what happens AFTER all articles are loaded.
+      app.Article.fetchAll();
```

Comment:
Note that fetchAll takes a parameter as it's callback. which you want it to be the initAdminPage.